docs/test-pr-guide.md

```diff
@@ -13,7 +13,7 @@
 5. Run the following commands to get to the PR branch, where `<contributor>` is the GitHub username of the contributor that submitted the PR:
```

Comment:
Although it's debatable, we can change it to something like
```suggestion
5. Run the following commands to get to the PR branch, where `<id>` is the PR id and `<branch_name>` is the random name for the branch.
```

CC:- @anitab-org/mentorship-maintainers

Reply:
I just use github cli so can't really help with this.

Reply:
@keshavgbpecdelhi please make the required changes